### DISCUSSION TITLE
session crashes when using SPICE : very reproducible

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#1055] [Q&A] [answered]
Howdy, folks. I'm using SPICE for remote access and getting very reproducible session crashes and I can usually trigger them with some SPICE window resizes within about 30-60 seconds or so. 

What's the best way to try to troubleshoot this? I've yet to find much about where regolith might be dumping logs. Something in `journalctl --user` perhaps?

**Comment by nrvale0** [comment]
Update: So I've switched certain SPICE sessions to using other desktop systems and I'm still seeing these crashes but less frequently. Regardless, it points to likely possibility that the RC is not Regolith itself. It would still be helpful to understand where to find relevant logs etc but I'm going to Close this one. If anyone has suggestions as to how to troubleshoot specifically in the context of Regolith please do update even after Close. 

thx

**Reply by kgilmer** [reply, marked as answer]
Hi @nrvale0 , yeah I would tail `/var/log/syslog` or `journalctl -f` and look for something w/ a crash or fatal error.
